x-pack/plugins/reporting/server/export_types/common/get_full_urls.test.ts

```diff
@@ -5,12 +5,12 @@
  */
 
 import { ReportingConfig } from '../../';
-import { ScheduledTaskParamsPNG } from '../png/types';
-import { ScheduledTaskParamsPDF } from '../printable_pdf/types';
```

Comment:
I had to rename this, because these interfaces are not used as the parameters that go in to scheduling with Task Manager. These interfaces are used as part of the source in the reporting document